includes/class-debug.php

```diff
@@ -20,9 +17,11 @@ class Debug {
 	 * Initialize the class, registering WordPress hooks.
 	 */
 	public static function init() {
-		if ( WP_DEBUG_LOG ) {
+		if ( \WP_DEBUG && \WP_DEBUG_LOG ) {
```

Comment:
I'd keep it at only checking for WP_DEBUG_LOG. WP_DEBUG, to me, generally suggests that there'll be front-end facing debug information. Logs could be collected without that